src/mailadm/bot.py

```diff
@@ -76,10 +75,10 @@ def ac_incoming_message(self, message: deltachat.Message):
                 logging.info("forwarding the message to a support group.")
                 self.forward_to_support_group(message)
 
-    def is_support_group(self, chat: deltachat.Chat):
+    def is_support_group(self, chat: deltachat.Chat) -> bool:
         """Checks whether the group was created by the bot."""
-        if chat.is_group():
-            return chat.get_messages()[0].get_sender_contact() == self.account.get_self_contact()
+        return chat.is_group() \
+                and chat.get_messages()[0].get_sender_contact() == self.account.get_self_contact()
```

Comment:
```suggestion
        chatstarter = chat.get_messages()[0].get_sender_contact()
        return chat.is_group() and chatstarter == self.account.get_self_contact()
```

Hm, this is a bit nicer I think? But that's just a style question I guess :shrug: no strong opinion